test/testLauncher.js

```diff
@@ -13,6 +13,7 @@ var endTest = function (evt) {
 
     // Show information about objects created/disposed
     aria.core.Timer.addCallback({
+        scope : Aria,
```

Comment:
In fact, when there is no scope and no arg (as in this case), we could use a simple call to `setTimeout`.